app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -877,15 +877,15 @@ def test_end_experiment(self):
 
     def test_launch_experiment_valid_with_preview_status(self):
         user_email = "user@example.com"
-        experiment = NimbusExperimentFactory.create(
-            status=NimbusExperiment.Status.PREVIEW,
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle=NimbusExperimentFactory.Lifecycles.PREVIEW
         )
         response = self.query(
             UPDATE_EXPERIMENT_MUTATION,
             variables={
                 "input": {
                     "id": experiment.id,
-                    "status": NimbusExperiment.Status.DRAFT.name,
+                    "status": NimbusExperiment.Status.PREVIEW.name,
```

Comment:
This is true. I removed `LIVE` from the allowed list for `PREVIEW`. Let's see if that breaks the integration tests 👀 (and make sure it still works)

Reply:
womp womp: looks like it was actually going from Preview to Live

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/43795363/206037215-bb315699-a251-4a87-9e4a-56183b388819.png">

The question is, do we want to update our diagrams, or do we want to update the code 🤔

Reply:
that's what I'm thinking too... @jaredlockhart ?

Reply:
Experiment is in preview:

<img width="319" alt="image" src="https://user-images.githubusercontent.com/119884/206253433-31a9d828-8c3c-4bf5-b37f-8cc2f9d4a9c6.png">

Click the launch button, request:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/119884/206254447-3b3379d1-f661-4907-9293-29b35de1c1d7.png">

Experiment is in draft/review(live)

<img width="290" alt="image" src="https://user-images.githubusercontent.com/119884/206253665-0a4b4bb3-b92c-47ce-9f22-71016470253f.png">

It doesn't go to live until after it's already approved in remote settings.

I don't understand why we'd be changing the launch flow at all in this PR?  Shouldn't this only be scoped to the live update changes?

Reply:
> womp womp: looks like it was actually going from Preview to Live
> 
> <img alt="image" width="1048" src="https://user-images.githubusercontent.com/43795363/206037215-bb315699-a251-4a87-9e4a-56183b388819.png">
> 
> The question is, do we want to update our diagrams, or do we want to update the code 🤔

Also this is talking about `status_next` not `status`

Reply:
The change you made in the PR is to go to `preview/review(live)` but that's not the right state, the docs and the code both describe `draft/review(live)`, which is the state it would go to either from `draft/idle(none)` or `preview/idle(none)`.   None of that should have changed?